kakao-ext-clicks/build.gradle.kts

```diff
@@ -1,31 +1,40 @@
 plugins {
     id("com.android.library")
-    id("kotlin-android")
-    id("com.github.ben-manes.versions") version Versions.gradle_versions
+    kotlin("android")
 }
 
 android {
-    compileSdk = 33
+    namespace = "io.github.kakaocup.kakao.ext.clicks"
+    compileSdk = 34
     defaultConfig {
         minSdk = 14
-        targetSdk = 33
         multiDexEnabled = true
     }
 
     sourceSets {
```

Comment:
Do we really still need sourceSets instructions here? this is the default behavior in AGP